clients/python/src/model_registry/types/artifacts.py

```diff
@@ -87,23 +121,25 @@ class DocArtifact(Artifact):
         external_id: Customizable ID. Has to be unique among instances of the same type.
     """
 
+    uri: str | None = None
+
     @override
-    def create(self, **kwargs) -> Any:
+    def create(self, **kwargs) -> DocArtifactCreate:
         """Create a new DocArtifactCreate object."""
         return DocArtifactCreate(
             customProperties=self._map_custom_properties(),
             **self._props_as_dict(exclude=("id", "custom_properties")),
-            artifactType="doc-artifact",
+            artifactType="model-artifact",
```

Comment:
```suggestion
            artifactType="doc-artifact",
```

I didn't understand why this change